topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga

```diff
@@ -0,0 +1,1116 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
```

Comment:
The 'annotation' attribute is missing. Please change to: 
```
"annotation": "<title of tutorial>"
```
 replacing <title of tutorial> with the title of the tutorial.

Reply:
@subinamehta This should be the last change to make to get the linter to pass 👍